content/en/observability_pipelines/setup/_index.md

```diff
@@ -393,6 +393,15 @@ For a deeper dive into recommendations for deploying and scaling multiple Worker
 - See [Deployment Design and Principles][7] for information on what to consider when designing your Observability Pipelines architecture.
 - See [Best Practices for OPW Aggregator Architecture][8] for details on the recommended Observability Pipelines aggregator architecture, which is optimized for scaling.
 
+## Deployment Modes
```

Comment:
I moved the info about requesting access to a banner: https://a.cl.ly/z8ud6qXY

What does Support have to do when they get a request? We have to add those instructions to [The Public Docs say Contact Support](https://datadoghq.atlassian.net/wiki/spaces/TS/pages/328404964/The+Public+Docs+say+Contact+Support).

```suggestion
## Deployment Modes

<div class="alert alert-warning">
  Remote configuration for Observability Pipelines is in private beta. Contact <a href="https://docs.datadoghq.com/help/">Datadog support</a> or your Customer Success Manager for access.
</div>
```

Reply:
i like this! 

As for your question, support has to reach out to me or @silversupreme to get the customer enrolled